sunpy/net/hek/utils.py

```diff
@@ -0,0 +1,197 @@
+import os
+import re
+import json
+from pathlib import Path
+
+from regions import PolygonSkyRegion
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+from sunpy.time import parse_time
+
```

Comment:
We will need to add an `__all__` to this file which will define the "api" for this file